docs/operate-and-deploy/installation/server-config/config-reference.md

```diff
@@ -612,11 +612,13 @@ property has the value `KSQL_PROCESSING_LOG`.
 Toggles whether or not the processing log should include rows in log
 messages. By default, this property has the value `false`.
 
-### ksql.logging.server.skipped.response.codes
+### ksql.logging.server.rate.limited.response.codes
 
-A comma-separated list of HTTP response codes to skip during server
-request logging. This is useful for ignoring certain 4XX errors that you
-might not want to show up in the logs.
+A list of `path:rate_limit` pairs, to limit the rate of server request
+logging. This is useful for limiting certain 4XX errors that you
+might not want to blow up in the logs. 
+This setting enables seeing the logs when the request rate is low 
+and dropping them when they go over the threshold.
```

Comment:
Mention that a message will be logged (at most once every five seconds) if the threshold is hit? (And same for the other config below.)